cleanlab/multilabel_classification/filter.py

```diff
@@ -0,0 +1,228 @@
+import warnings
+from typing import Optional, Union, Tuple, List, Any
+import numpy as np
+
+
+def find_label_issues(
```

Comment:
I think you have copy/paste errors between docstring for this method and docstring for: `find_multilabel_issues_per_class`

Reply:
Fixed: https://github.com/cleanlab/cleanlab/blob/64f6d6b648758107bac56681fef3a387b4ccd548/cleanlab/multilabel_classification/filter.py#L135